### DISCUSSION TITLE
I have installed Open-WebUI locally, but when adding a model, there are no search results, and the list is empty.

### DISCUSSION THREAD

**Discussion by tpjaord1983** [open-webui/open-webui#7600] [General]
![微信截图_20241204174455](https://github.com/user-attachments/assets/c623b4c5-860c-4d59-ba34-0e3df99ddca4)

**Comment by juananpe** [comment]
Have a look at /admin/settings. Have you set up a correct OpenAI API key there?
<img width="1232" alt="image" src="https://github.com/user-attachments/assets/7d2e7fa5-8718-4d39-8afe-eb3dcbed66c3">
Click on the gear icon and try to verify (reload) the connection:
![image](https://github.com/user-attachments/assets/b20137a5-5dbd-4440-9627-538ef1164801)
you should get a success message like this:
<img width="394" alt="image" src="https://github.com/user-attachments/assets/e2c330e1-d969-4985-a673-e606efaea1eb">
